lib/ecto_vista.ex

```diff
@@ -2,65 +2,6 @@ defmodule EctoVista do
   @moduledoc """
   Provides the macros and functions to define and manage
   PostgreSQL views with Ecto.
-
-  ## Using EctoVista
```

Comment:
I was just thinking again about it and come to conclusion that I would prefer more meaningful changes than just moving from one file to another.
If you don't mind I'll revert the docs part of your PR